tests/test_param_substitution.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+
+"""
+<Program Name>
+  test_verifylib.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Nov 07, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test verifylib functions.
```

Comment:
This purpose seems wrong.  It's not really testing the verifylib functions, but only param substitution.  There already is a test_verifylib.py.  Shouldn't the substitute function be tested there, in `test_verifylib.py`?